src/questions/install-command.spec.js

```diff
@@ -11,4 +12,16 @@ describe('askInstallCommand', () => {
       default: 'npm install'
     })
   })
+
+  it('should return correct question format when project lang is js', () => {
```

Comment:
You put the same description test than the previous test.
It should be: 'should return correct question format when project lang is **not** js'

Reply:
Nice catch 👍